client/src/components/authentication/SignInForm.js

```diff
@@ -103,7 +104,7 @@ const SignInForm = () => {
           </label>
         </div>
         <div>
-          <input type="submit" className="button" value="Sign In" />
+          <input type="submit" className="button sign-up-button" value="Sign In" />
```

Comment:
you are using this class of `sign-up-button`, but in multiple places, including this sign in form. You need to use a more generalized name in this case, like "authentication-button"